src/main/java/duke/bot/Ui.java

```diff
@@ -0,0 +1,120 @@
+package duke.bot;
+
+import java.util.List;
+
+import duke.task.Task;
+
+/** An object that handles the printing of chat bot's messages*/
+public class Ui {
+    /**
+     * Constructs a message
+     *
+     * @param msg Message to be displayed
+     * @return A formatted string containing the constructed message
+     */
+    public String constructMsg(String msg) {
```

Comment:
Maybe using constructMessage sounds better and more fluent than using constructMsg?